[MonorepoBuilder] Impossible to register custom ReleaseWorker

### The problem
Trying to register a custom ReleaseWorker with `$services->set(App\MyCustomReleaseWorker::class);` results in:
```bash
> vendor/bin/monorepo-builder release major --dry-run

                                                                                                                        
 [ERROR] There are no release workers registered. Be sure to add them to "monorepo-builder.php"                         
                                                                                                                        
```

I found that [line 27](https://github.com/symplify/symplify/blob/e1be40027f00acf1f2aee885a36e82db62f3fcb4/packages/monorepo-builder/bin/monorepo-builder.php#L27) of the following code is breaking before requiring the `autoload.php` from the "dependency" entry as soon as the "after split package" one is found.

https://github.com/symplify/symplify/blob/e1be40027f00acf1f2aee885a36e82db62f3fcb4/packages/monorepo-builder/bin/monorepo-builder.php#L14-L29

I think that this is blocking MonorepoBuilder from knowing anything about the code available inside the main project.

### Possible solution
Removing the `break;` line solves my issue, but I'm not aware of possible side-effects. Do you think that this can be a valid fix?

If so, then I can open a PR.

Thank you for your time.

--- a/packages/monorepo-builder/bin/monorepo-builder.php
+++ b/packages/monorepo-builder/bin/monorepo-builder.php
@@ -13,12 +13,12 @@ use Symplify\SymplifyKernel\ValueObject\KernelBootAndApplicationRun;
 
 # 1. autoload
 $possibleAutoloadPaths = [
+    // monorepo
+    __DIR__ . '/../../../vendor/autoload.php',
     // after split package
     __DIR__ . '/../vendor/autoload.php',
     // dependency
     __DIR__ . '/../../../autoload.php',
-    // monorepo
-    __DIR__ . '/../../../vendor/autoload.php',
 ];
 
 foreach ($possibleAutoloadPaths as $possibleAutoloadPath) {

